release/train_tests/benchmark/train_benchmark.py

```diff
@@ -142,11 +151,18 @@ def get_next_batch(self, dataloader):
 
     def train_step(self, input_batch, labels):
         self.model.train()
+        self.optimizer.zero_grad()  # 1. Clear FIRST
```

Comment:
Did it matter to clear first vs at the end?

Reply:
I don't think it matters. but we found that in the old benchmark, zero_grad was wrongly put in between forward and backward passes.